### PR TITLE
WIP: Physics day abstract

### PR DESCRIPTION
**Pull request by samumantha**

### CODE REVIEW

File: conference_abstracts/UH_physics_day24.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Track: Education and outreach 
+
+Title: CodeRefinery - supporting your research software development journey
+
+Draft:
+
+Reproducibility and openness is becoming of ever more crucial importance in
+any research ecosystem. In physics as in many other domains, research often
+relies on software written as a by-product, and it is an essential part of the
+researche workflow. However, software is often overlooked from the
```
Comment (bast) [inline conference_abstracts/UH_physics_day24.md]:
```suggestion
research workflow. However, software is often overlooked from the
```

**Comment by bast** [comment]
I will squash-merge since this seems to be a PR from an outdated branch.

**Comment by bast** [comment]
> I will squash-merge since this seems to be a PR from an outdated branch.

Ah I misunderstood. It's all good. I thought the PR earlier was towards main.

**Comment by bast** [comment]
Sorry only now I see the "WIP" ...

**Comment by samumantha** [comment]
No worries. We can still fix if needed. Thanks for merge :)

**Pull request merged**